compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/SchemaGenerationTest.java

```diff
@@ -193,6 +197,38 @@ public void testGraphqlServiceWithHttpService() {
         Assert.assertEquals(diagnosticResult.errorCount(), 0);
     }
 
+    @Test
+    public void testGeneratedGraphQLSDLSchema() {
```

Comment:
When using abbreviations the convention is only to capitalize the first letter.

```suggestion
    public void testGeneratedGraphqlSdlSchema() {
```